### PR TITLE
Issue #93: upgrade to checkstyle 7.6.1

### PR DESCRIPTION
**Pull request by romani**
Issue #93

### CODE REVIEW

File: checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml [hunk 1]
```diff
@@ -1245,6 +1245,10 @@
     <name><![CDATA[Package Declaration]]></name>
     <configKey><![CDATA[Checker/TreeWalker/PackageDeclaration]]></configKey>
     <status>READY</status>
+    <cardinality>MULTIPLE</cardinality>
```
Comment (rnveach) [inline checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml]: cardinality shouldn't be multiple.
We only have 1 property that is either true/false for this check. Users won't activate 2 instances with both values on.

Comment (romani) [inline checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml]: I missed your point, and did a release.
lets keep it as is , I will be more accurate/attentive later on.

**Comment by romani** [comment]
merged manually.

File: checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml (hunk 1 above)
Comment (rnveach) [inline checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml]: Will be fixed in cardinality UT issue.

**Pull request closed without merging**